openslides/assignments/static/js/assignments/site.js

```diff
@@ -200,8 +228,16 @@ angular.module('OpenSlidesApp.assignments.site', ['OpenSlidesApp.assignments'])
                 assignment.description,
                 $scope.phases[assignment.phase].display_name,
                 _.map(assignment.assignment_related_users,
-                        function (candidate) {return candidate.user.get_short_name();}).join(" "),
+                        function (candidate) {return candidate.user.get_short_name();}
+                ).join(" "),
+                _.map(
+                    assignment.tags,
+                    function (tag) {
+                        return tag.name;
+                    }
+                ).join(" "),
```

Comment:
Please use consistent code formatting / line breaks (as for line `_map(assignment.assign,ent_related_users...`)